tests/XRootD/http.sh

```diff
@@ -75,5 +75,66 @@ function test_http() {
 	       assert davix-rm "${HOST}/${TMPDIR}/${i}.ref"
 	done
 
+  # GET range-request
+	## Upload a file with a fixed content string
+	alphabetFile="alphabet.txt"
+	alphabetFilePath="${TMPDIR}/$alphabetFile"
+	echo -n "abcdefghijklmnopqrstuvw987" > $alphabetFilePath
+	assert curl -v -L -H 'Transfer-Encoding: chunked' "${HOST}/$alphabetFilePath" --upload-file $alphabetFilePath
+  outputFilePath=${TMPDIR}/output.txt
```

Comment:
Indentation looks a bit funny here. Also, could you please check the script with `shellcheck`? I think we can also avoid repetition by adding an extra assert function that checks two strings for equality (like ASSERT_EQ from Google Test, but for bash).

Reply:
very weird, I don't see this indentation issue on my IDE...

Reply:
Yes, OK for `assert_eq()` :+1:

Reply:
DOne